04_numeronym/paraganz/main_test.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var tests = []struct {
+	in  string
+	out []string
+}{
+	{"accessibility", []string{"a11y"}},
+	{"accessibility testing", []string{"a19g"}},
+	{"abc", []string{"abc"}},
+	{"a", []string{"a"}},
+	{"", []string{""}},
+	{" ", []string{" "}},
```

Comment:
Add unicode tests too.

Reply:
Done

Reply:
Sorry to be a pain, but I think if you add a test with unicode at beginning/end this will fail.
Also please add tests with whitespace at beginning/end/middle - It probably should be trimmed with `strings.Trim(str, "\n\t ")`.

Reply:
Done. Added test for mentioned sceanrios